MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -170,7 +178,9 @@ class RouteManeuverViewController: UIViewController {
     }
     
     func updateStreetNameForStep() {
-        if let destinations = step?.destinations {
+        if let currentLeg = leg, let step = step, step.maneuverType == .arrive {
+            destinationLabel.unabridgedText = currentLeg.name
```

Comment:
The label should be set to the destination’s name, not the leg’s name. Is this why you were seeing two street names separated by a comma in https://github.com/mapbox/mapbox-navigation-ios/pull/419#issuecomment-317906512?